blocks.md

```diff
@@ -0,0 +1,111 @@
+
+The namespace prefix to the block name should reflect either the plugin author, or the plugin slug. For example:
+
+`name: "my-rainbow-block-plugin/rainbow"`, or
+`name: "john-doe/rainbow"`, or
+`name: "pluginco/rainbow"`.
+
+The namespace must not be a reserved one such as `core`.
+
+<h4>4. Block Plugins must include a <code>block.json</code> file.</h4>
+
+The Block Registration RFC outlines the format of a `block.json` file: https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md
+
+Block Plugins must include a valid `block.json` file. In addition to the requirements specified in the RFC, the `block.json` must include the following attributes:
+* name
+* title
+* At least one of: script, editorScript
+* At least one of: style, editorStyle
+
+<h4>5. Block Plugins must not require payment or a paid account to function.</h4>
+
+This includes payment for premium features. You may use the donation link feature.
+
+Block plugins that do require a paid service or include upselling and premium features are still permitted in the main WordPress Plugin Directory, subject to its guidelines.
+
+<h4>6. Block Plugins should work seamlessly.</h4>
+
+Block Plugins are intended to work seamlessly and instantly when installed from the editor. That means they should not encumber the user with additional steps or prerequisites such as installing another plugin or theme, signing up for an account, or logging in or manually connecting to an external service.
+
+Block Plugins must not require another plugin in order to function.
+
+They may use an external/cloud API where necessary, provided it can be done seamlessly without requiring a login or activation key.
+
+They should not rely on an external API or cloud service for functions that could be performed locally.
+
+<h4>7. Server-side code should be kept to a minimum.</h4>
+
+Since Block Plugins are WordPress plugins, they necessarily contain PHP code for metadata and initialisation. As much as possible, they should avoid including additional PHP code or server-side libraries. The WordPress REST API should be the preferred interface to WordPress, rather than custom server-side code.
+
+We recognise that there are limits to the REST API, and situations where server-side PHP is the only performant way to implement a feature. In those situations, PHP code may be included. Please ensure that such code is clearly written, used as little as possible, and well documented.
+
+<h4>8. Must not include advertisements or promote other blocks, plugins, or themes.</h4>
+
+Block Plugins must not include code that displays alerts, dashboard notifications, or similar messages promoting other plugins, themes, goods, or services. They may include an unobtrusive link identifying the developer or plugin page.
```

Comment:
"They may include an unobtrusive link identifying the developer or plugin page." 

Okay I have to ask... How? Because you can't have thee link on the front facing side, so are we saying using the block UX (sidebar) is okay for mild self promotion or...?

Reply:
I've totally rewritten this section.